tests/testsuite/rustdoc.rs

```diff
@@ -39,6 +39,19 @@ fn rustdoc_args() {
         .run();
 }
 
+#[cargo_test]
+fn rustdoc_binary_args_passed() {
+    let p = project().file("src/main.rs", "").build();
+
+    p.cargo("rustdoc -v")
+        .arg("--")
+        .arg("--theme")
+        .arg("dark")
```

Comment:
Can this use a stable option that doesn't fail?  Maybe `--markdown-no-toc`, it seems pretty simple?